spark/src/test/scala/org/apache/mahout/sparkbindings/test/DistributedSparkSuite.scala

```diff
@@ -39,6 +39,7 @@ trait DistributedSparkSuite extends DistributedMahoutSuite with LoggerConfigurat
           .set("spark.kryoserializer.buffer.mb", "15")
           .set("spark.akka.frameSize", "30")
           .set("spark.default.parallelism", "10")
+          .set("spark.driver.allowMultipleContexts", "true")
```

Comment:
This would seem to imply some tests are nesting contexts. If so it's a Mahout bug. 

Spark 1.2 is probably the next thing to target since it's been out for awhile and is in the distros. It also has the fix for missing anon functions in the artifacts so it should remove the need to build Spark from source.